### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by katrogan**
# TL;DR
https://github.com/lyft/flytepropeller/pull/187 adds a check to flytepropeller so that a workflow lacking nodes fails during registration/compilation time. Admin integration tests use incomplete workflow skeletons that do not include any nodes and therefore fail.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Review by wild-endeavor — DISMISSED** [review]

**Review by EngHabu — DISMISSED** [review]

**Comment by katrogan** [comment]
@EngHabu can I get one last review please?

**Comment by anandswaminathan** [comment]
+1
